sources/controllers/node_controller.rb

```diff
@@ -11,28 +9,35 @@ class NodeController < Sinatra::Base
   end
 
   # html view, sends stuff to /node as POST
-  get '/new' do
-    @node = Node.new
-    haml :node
-  end
-
-  post '/' do
-    Node.new(params[:node])
-  end
+  get '/new' do             #
+    @node = Node.new        # dafuq is this
+    haml :node              #
+  end                       #
 
-  delete '/:id' do
-    Node.delete(params[:id])
-  end
+  post '/' do               #
+    Node.new(params[:node]) # dafuq is that
```

Comment:
why to you have logical duplicants? get on the /new and post on / should do the same thing. perform a GET request on /new to create a new node should not be part of a REST-API